vectorization/README.md

```diff
@@ -0,0 +1,149 @@
+# Vectorization
+
+This sample shows how to implement matrix multiplication using various
+vectorization approaches.
+
+Note: You should not reuse this matrix library in your application. It was not
+written to be useful beyond the scope of this demo. If you're looking for a
+matrix library, you probably want [GLM] for graphics applications, or a linear
+algebra library for compute applications.
```

Comment:
"such as BLAS"?

Reply:
I only know a couple by name and wasn't sure if any of them were actually good :)